### DISCUSSION TITLE
Posters + Fan Art

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#1125] [Art]
Moved from Github issue https://github.com/sunflower-land/sunflower-land/issues/1071

## Summary

Posters, promotional banners, and fan-art.

The purpose of this request is to gather fan art that we can display in our Sunflower Studios during live streams. We would like to rotate fan art on a weekly basis and give shoutouts during our live streaming of the development of the game.

## Requirements

- Minified art submission
- Original files (aseprite, illustrator etc.)
- Aspect ratio for posters

You have read the contributing guidelines:
https://github.com/sunflower-land/sunflower-land/blob/main/docs/ART_CONTRIBUTING.md

You agree to the project's terms and conditions:
https://docs.sunflower-land.com/support/terms-of-service

Note: Your art & mechanic may be modified/altered by the community if needed.

## Rewards

There is no guarantee of rewards by contributing to the project. Based on past history however, artists are airdropped their designs or surprise with in game rewards 😉 

## Deadline

No strict deadline on this. We would like to do a live stream on Tuesday so looking for some initial art work to showcase and shoutout for the community ❤️

**Comment by On1el** [comment]
I tried to make a poster for the streamers with one villain in the title rol
![Плакат](https://user-images.githubusercontent.com/104461938/175788075-1208dd42-ab42-4314-a29b-47b94a01e291.png)

**Comment by digital-farms** [comment]
![логоКЧ](https://user-images.githubusercontent.com/105886943/176555545-64991f45-e1b5-4af6-9182-7f64c3e12dfe.png)
![InShot_20220628_113737509](https://user-images.githubusercontent.com/105886943/176555552-2acad46a-17c2-4625-a804-36095203d98a.jpg)

**Comment by JcEii** [comment]
### SFL Cockfight by Jc Eii (Ft. Esp123)

Rooster is up today.

_Therefore..._

**It's Cockfight Time "San ka? Talpakan na!"**  

**Art:**
![Sabungan](https://user-images.githubusercontent.com/105348036/177170422-fb4c7544-ce07-41c3-97ef-6befb2a81888.gif)

**Aseprite and icons:**
[Cockfight Art Aseprite  by Jc Eii (Ft. Esp123).zip](https://github.com/sunflower-land/sunflower-land/files/9040020/Cockfight.Art.Aseprite.by.Jc.Eii.Ft.Esp123.zip)

**Aspect ratio:** 1:1 (800x800)

**Reply by adamhannigan** [reply]
Wow that chicken is jacked! 😮

**Comment by On1el** [comment]
![SHARK](https://user-images.githubusercontent.com/104461938/178359774-74e4ee97-8f0e-42d2-a880-92f62561b9da.png)

**Comment by On1el** [comment]
Greedy goblins got what they deserved
![RG](https://user-images.githubusercontent.com/104461938/180665746-6f1ce9d7-38a5-4dd1-a03e-aec9f037e35c.gif)

**Comment by On1el** [comment]
![sp](https://user-images.githubusercontent.com/104461938/181014299-a73db340-16d7-49da-bc27-2099fc86b9c3.png)

**Comment by sfwhat** [comment]
![landex22222](https://user-images.githubusercontent.com/95937808/183252037-cec27c46-4c5f-42fa-8b81-0e1ccfcfd2f8.png)
1x1 ![landexxxxlogo](https://user-images.githubusercontent.com/95937808/183252056-90fa389c-3996-4782-9167-4d1728a2cc76.png)

![SfWhat](https://user-images.githubusercontent.com/95937808/183252030-4fe61968-f923-4cba-948e-849a3dbc0143.gif)

**Comment by NycksonV** [comment]
I found the game's Boss very cool but I think something is missing, so I decided to make a worthy animation for 'Lord Litch'

Achei o chefe do jogo muito legal, mas acho que algo está faltando, então decidi fazer uma animação digna para 'Lord Litch''

![Lord](https://user-images.githubusercontent.com/118012559/201411639-f30fc7a7-2bc4-4014-8458-f7230f2cce4c.gif)

![Lord](https://user-images.githubusercontent.com/118012559/201410635-69ff04f4-ff13-4147-88a9-cace9eb6547c.gif)



I'm from the farm:136998

Hope you like it ^^

**Reply by digital-farms** [reply]
No one has ever mocked my king like that 🤣 🤣 🤣
Great))

**Reply by diasfigor** [reply]
Muito bom. Parabéns.

**Reply by digital-farms** [reply]
> Muito bom. Parabéns.

I love it, I love it)
Sorry, language barrier(

**Reply by NycksonV** [reply]
thanks
